web/frontend/templates/index.html

```diff
@@ -1,20 +1,18 @@
-{% load render_bundle from webpack_loader %}
-{% load static %}
+{% load render_bundle from webpack_loader %} {% load static %}
```

Comment:
Django templates should be ignored because the linter doesn't understand them properly. It still thinks it's all HTML even though there is some template language stuff in it too. It'd probably be fine to just ignore everything in _any_ directory named "templates". Pretty sure exclusions for the hook can be specified in the pre-commit yaml file.

Also, please put these back to being on separate lines like they were before. Also do this for lines 15 and 16.